Automattic/jetpack
# Jetpack Recommendations: Optimize images

Fixes #18587

#### Changes proposed in this Pull Request:


This PR replaces the .png images in the Recommendations tab with optimized .jpg versions. This is done to improve load time.

#### Jetpack product discussion


p8oabR-BT-p2#comment-4905

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






1. Visit `/wp-admin/admin.php?page=jetpack#/recommendations/site-type`.
2. Advance through the prompts. Be sure to select "Store" so that you see the WooCommerce prompt.
3. Verify that images quickly load for each prompt.

#### Proposed changelog entry for your changes:


N/A
